model/src/main/java/io/gravitee/definition/model/Rule.java

```diff
@@ -15,21 +15,25 @@
  */
 package io.gravitee.definition.model;
 
-import com.fasterxml.jackson.annotation.*;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import io.gravitee.common.http.HttpMethod;
+
 import java.io.Serializable;
-import java.util.*;
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.Set;
 
 /**
  * @author David BRASSELY (david.brassely at graviteesource.com)
  * @author GraviteeSource Team
  */
-public class Rule implements Serializable {
+public class Rule extends HashMap<String, Object> /* This is to generate the correct Open-API definition*/implements Serializable {
```

Comment:
This won't be the right schema, since the maps'

- key matches `Policy::name`
- value matches `Policy::configuration`

This kind of schema is very suboptimal, but now we need to use it b/c of backwards compatibility

Reply:
🙄  you're right !
Forget my comment ! 😉